share/picongpu/pypicongpu/template/etc/picongpu/N.cfg.mustache

```diff
@@ -60,15 +61,13 @@ TBG_wallTime="1:00:00"
 
 TBG_steps="{{{time_steps}}}"
 
-{{#grid.boundary_condition}}
-  TBG_periodic="--periodic {{{x}}} {{{y}}} {{{z}}}"
-{{/grid.boundary_condition}}
-
-
 #################################
 ## Section: Optional Variables ##
 #################################
 
+{{#grid.boundary_condition}}
+  TBG_periodic="--periodic {{{x}}} {{{y}}} {{{z}}}"
+{{/grid.boundary_condition}}
```

Comment:
why have these been moved under optional variables?

Reply:
because they are considered optional by the PIConGPU parsing of run time parameters.

PIConGPU uses the default  set in the .param files if periodic boundaries are not specified.

In contrast PIConGPU  will not start the simulation if grid size or device number is not specified.

I therefore opted to move them to the optional section while reworking the template, doesn't change anything function wise, just groups stuff correctly.